packages/workflow/src/Extensions/ArrayExtensions.ts

```diff
@@ -364,27 +356,130 @@ function intersection(value: unknown[], extraArgs: unknown[][]): unknown[] {
 	return unique(newArr, []);
 }
 
+average.doc = {
+	name: 'average',
+	description: 'Returns the mean average of all values in the array',
+	returnType: 'number',
+};
+
+compact.doc = {
+	name: 'compact',
+	description: 'Removes all empty values from the array',
+	returnType: 'array',
+};
+
+isEmpty.doc = {
+	name: 'isEmpty',
+	description: 'Checks if the array doesn’t have any elements',
+	returnType: 'boolean',
+};
+
+isNotEmpty.doc = {
+	name: 'isNotEmpty',
+	description: 'Checks if the array has elements',
+	returnType: 'boolean',
+};
+
+first.doc = {
+	name: 'first',
+	description: 'Returns the first element of the array',
+	returnType: 'array item',
+};
+
+last.doc = {
+	name: 'last',
+	description: 'Returns the last element of the array',
+	returnType: 'array item',
+};
+
+max.doc = {
+	name: 'max',
+	description: 'Gets the maximum value from a number-only array',
+	returnType: 'number',
+};
+
+min.doc = {
+	name: 'min',
+	description: 'Gets the minimum value from a number-only array',
+	returnType: 'number',
+};
+
+randomItem.doc = {
+	name: 'randomItem',
+	description: 'Returns a random element from an array',
+	returnType: 'number',
+};
+
+unique.doc = {
+	name: 'unique',
+	description: 'Returns a random element from an array',
```

Comment:
Description not correct.

Reply:
Turns out this takes optional args, so moved to next phase.